app/src/main/java/com/randomappsinc/studentpicker/grouping/GroupingFragment.java

```diff
@@ -0,0 +1,87 @@
+package com.randomappsinc.studentpicker.grouping;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.joanzapata.iconify.fonts.IoniconsIcons;
+import com.randomappsinc.studentpicker.R;
+import com.randomappsinc.studentpicker.utils.UIUtils;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.Unbinder;
+
+public class GroupingFragment extends Fragment {
```

Comment:
I've changed the name of the GroupFragment to GroupingFragment

Reply:
nit: How about GroupMakingFragment? I feel like that's more declarative.

Reply:
yes, that is better.